### DISCUSSION TITLE
What does 'other sync service' mean in the sync methods

### DISCUSSION THREAD

**Discussion by thibaultmol** [Vinzent03/obsidian-git#273] [Q&A]
I don't really understand it nor do I find documentation on it anywhere
(and I guess in general giving users a clearer understand of each mode, maybe in the readme of this plugin, would be nice)

**Comment by Vinzent03** [comment]
It's basically already described with `Other sync service (Only updates the HEAD without touching the working directory)`. It updates the `HEAD` to the latest commit, but doesn't change any files. This is useful when syncing e.g. with Obsidian sync, but want a git repo in addition. Files get synced via Obsidian sync and the HEAD via Obsidian Git.

**Comment by ricardonevesmusic** [comment]
> It's basically already described with `Other sync service (Only updates the HEAD without touching the working directory)`. It updates the `HEAD` to the latest commit, but doesn't change any files. This is useful when syncing e.g. with Obsidian sync, but want a git repo in addition. Files get synced via Obsidian sync and the HEAD via Obsidian Git.

Hi @Vinzent03

Could you please explain it further?

1. What does enabling this option makes or means in practical terms?

2. I don't know if you are saying that files don't get pushed/uploaded (and/or updated) to the repo or if are saying that it just keeps track of what was changed in a file (basically a commit/change or file version/history).

3. Basically, I guess you're saying enabling this option won't do what using Obsidian Sync or Syncthing will (correct me if I'm wrong).

4. Let's say I have Syncthing (or whatever sync service/app/plugin I'm using that sends/receives files on both ends): then what should I do? What option should I select?

Hope you can help.

Thank you.

**Reply by Vinzent03** [reply]
1. Again as described above it only sets the HEAD to the latest commit when pulling
2. It pushes normally, but doesn't pull any changes from remote, as this should be done by the other sync service
3. Parially, the sync to e.g. GitHub is still working, but not from GitHub to Obsidian.`
4. If you want to also keep a history of your changes on GitHub, use the "Other Sync Service" option, otherwise you don't need the plugin at all.
